src/common/utility.ts

```diff
@@ -267,32 +267,41 @@ export class Utility {
         const moduleDirs: string[] = await Utility.getSubDirectories(modulesPath);
         await Promise.all(
             moduleDirs.map(async (module) => {
-                await this.setModuleMap(module, moduleToImageMap, imageToDockerfileMap, imageToBuildOptions);
+                await this.setModuleMap(module, moduleToImageMap, imageToBuildSettings);
             }),
         );
     }
 
+    public static getBuildSettings(
+        modulePath: string,
+        dockerFilePath: string,
+        buildOptions?: string[],
+        contextPath?: string): BuildSettings {
+        const optionArray = (buildOptions && buildOptions instanceof Array) ? buildOptions : undefined;
+        const context = contextPath ? path.join(modulePath, contextPath) : path.dirname(dockerFilePath);

```

Comment:
I think `path.resolve` is better than `path.join` here as `path.resolve` can handle the case where `contextPath` is absolute.

Reply:
good catch, will update